generator/template_repo.go

```diff
@@ -285,6 +284,29 @@ func (t *Repository) LoadDir(templatePath string) error {
 	return nil
 }
 
+// LoadContrib loads template from contrib directory
```

Comment:
Do you really need this?
Afaik, dependencies are managed automatically by the template repo.
This works when there are external files, I did not check with preloaded assets.

Reply:
@fredbi this function replaces the "default" template by the one defined in the "contrib/<stratoscale>" directory.
The "contrib/<stratoscale>" was added to the bindata.go file, but they only replace the default files when the --template flag is set.
You may suggest another way to switch the default file and we will implement it.

Reply:
What I mean is why don't you just mimick what we do manually with the yaml config file?

When using the config file, I change the path of my templates and everything which needs override is well resolved without a single change in the repo_template code.

If there is a way to do that from config rather than altering this code, all the better, don't you think?

Reply:
Consider applying overrides here: https://github.com/go-swagger/go-swagger/blob/8914a0f9bc4c11970552f133f8c9af106c1ac511/generator/shared.go#L222